### PR TITLE
feat(catalog-backend-module-gitlab): Added option to skip forked repos

### PR DESCRIPTION
**Pull request by alexmanno**
## Hey, I just made a Pull Request!

Added an option in the gitlab catalog plugin to skip importing projects that are forks

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by mikeyhc — APPROVED** [review]

**Review by mikeyhc — APPROVED** [review]

**Comment by alexmanno** [comment]
We would be very interested in having this feature
How can I help you speed up the review process?
@mikeyhc

**Comment by alexmanno** [comment]
@mikeyhc can we re-open it?

**Comment by alexmanno** [comment]
@Rugvip are there any chances that this pr will be reopened?

**Comment by alexmanno** [comment]
comment the PR to remove the stale

File: .changeset/silent-garlics-drum.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
```
Comment (Rugvip) [inline .changeset/silent-garlics-drum.md:2]:
```suggestion
'@backstage/plugin-catalog-backend-module-gitlab': patch
```

**Review by Rugvip — APPROVED** [review]
Thank you, looks good! 🎉 

Switching the changeset to a `patch` bump since the module package is still on 0.x and this isn't breaking.

And sorry not sure how, but this didn't get picked up by GitHub review assignment automation so had to discover it myself. Will keep an eye out for if this happens more or whether this was a hiccup

**Pull request merged**